app/services/stripe_sessions_creator.rb

```diff
@@ -1,14 +1,16 @@
 class StripeSessionsCreator
 
-  def initialize(product_name, amount, quantity, host)
+  def initialize(product_name, product_image_url, amount, quantity, host)
```

Comment:
```suggestion
  def initialize(product_name:, product_image_url:, amount:, quantity:, host:)
```
What do you think of that @DavidGeismarLtd ?

Reply:
why not. For me `product_name:, product_image_url:, amount:, quantity:,` are all linked to the same concept : a `SoldItem` or `CartItem` for instance so I would have done something like` initialize(sold_item, host)` where sold_item is a SoldItem with` product_name:, product_image_url:, amount:, quantity:` accessors

Reply:
You are totally bro. I will refactor it. Thanks for the feedback.